test/run.sh

```diff
@@ -57,13 +57,21 @@ prepare() {
 
 run_test_application() {
   echo "Starting test application ${APP_IMAGE}..."
-  docker run --rm --cidfile=${cid_file} -p ${test_port}:${test_port} $1 ${APP_IMAGE}
```

Comment:
Why remove the `--rm` flag? Am I missing someplace else where the test container is removed?

Reply:
If we don't as soon as the container is stopped it will be removed and we won't be able to inspect the exit status which now want to check.